openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -175,95 +175,102 @@ The following subsections illustrate some of the authorization flows supported b
+(1a) The Wallet-initiated flow begins as the user requests a credential from a pre-configured list in the Wallet or as the user requires a credential for a presentation not available in his Wallet and the Wallet guides him to get a such a credential from a Credential Issuer.
 
-(2) The Wallet sends an Authorization Request to the Authorization Endpoint. The Authorization Endpoint processes the Authorization Request, which typically includes End-User authentication and gathering of End-User consent. 
+(1b) The Issuer-initiated flow begins as the Credential Issuer generates a Credential Offer for certain Credential(s) and communicates it to the Wallet, for example, as a QR code or as a link. The Credential Offer contains the Credential Issuer's URL and the credential type(s).
 
-(3) The Authorization Endpoint returns an Authorization Response with the Authorization Code upon successfully processing the Authorization Request. 
+(2) The Wallet uses the Credential Issuer's URL to fetch its metadata as described in (#credential-issuer-metadata) to learn what credential types and formats the Credential Issuer supports and to determine the Authorization (OAuth 2.0 Authorization Server) and Credential Endpoints. This specification enables deployments where the Credential Issuer API and the Authorization Server are different services, perhaps even provided by different entities. Note in this example, the Credential Issuer also provides the OAuth 2.0 Authorization Server.
 
-Note: Steps (2) and (3) happen in the front channel, by redirecting the End-User via the User Agent. Those steps are defined in (#authorization_endpoint).
+(3) The Wallet sends an Authorization Request to the Authorization Endpoint. The Authorization Endpoint processes the Authorization Request, which typically includes End-User authentication and gathering of End-User consent.
 
-(4) The Wallet sends a Token Request to the Token Endpoint with the Authorization Code obtained in step (3). The Token Endpoint returns an Access Token in the Token Response upon successfully validating Authorization Code. This step happens in the back-channel communication (direct communication between two systems using HTTP requests and responses without using redirects through an intermediary such as a browser). This step is defined in (#token_endpoint).
+Note: Steps (3) and (4) happen in the front channel, by redirecting the End-User via the User Agent. Those steps are defined in (#authorization_endpoint). The Authorization Server and the User Agent may exchange any further communication, the actual user authentication is out of scope for this specification.
```

Comment:
```suggestion
Note: Steps (3) and (4) happen in the front channel, by redirecting the End-User via the User Agent. Those steps are defined in (#authorization_endpoint). Since the End-User authentication is out of scope for this specification, the Authorization Server and the User Agent may exchange any further communication.
```